### ISSUE TITLE
Question about the probability

### ISSUE THREAD

**Issue by lizhiyuanUSTC** [nilboy/tensorflow-yolo#31]
Hi, thanks for your great work.
I notice that the output of the yolo is a 7*7*30 tensor, and some of the output means Pr(class|object), and some means Pr(object)， but these output may be <0, so they don't means probability any more. 
How to understand these problems?

**Comment by kmonachopoulos** [comment]
Pass them through a sigmoid function .. maybe that will help!

**Comment by lizhiyuanUSTC** [comment]
I am trying to train a model with softmax and sigmoid, but the result is terrible...
